docs/cli-help.txt

```diff
@@ -479,10 +479,9 @@ apache:
   Apache Web Server plugin
 
   --apache-enmod APACHE_ENMOD
-                        Path to the Apache 'a2enmod' binary (default: a2enmod)
```

Comment:
What's going on here?

Reply:
Is this a Mac thing?

Reply:
Yeah. It's the Apache version of the problem we used to see with Nginx before you wrote https://github.com/certbot/certbot/pull/6137. Joona is working on fixing this for Apache in #6598.